stable/spinnaker/templates/statefulsets/halyard.yaml

```diff
@@ -143,7 +143,9 @@ spec:
   - metadata:
       name: halyard-home
       labels:
-{{ include "spinnaker.standard-labels" . | indent 8 }}
+        app: {{ include "spinnaker.fullname" . | quote }}
```

Comment:
Would it be better to place these values in a helper, then reference it to be something as follows?
`{{ include "spinnaker.standard-labels-base" . | indent 8 }}`

I know it is another layer of abstraction, but we can use the base labels for other parts of the chart where necessary.

Reply:
@dwardu89 Thanks for your review.
The commit 50efbac defines "spinnaker.standard-labels-base", and use it.